pkg/resourcegroup/runaway/checker.go

```diff
@@ -275,7 +275,7 @@ func (r *Checker) markRunawayByWatch(action rmpb.RunawayAction) {
 func (r *Checker) markRunaway(matchType string, action rmpb.RunawayAction, now *time.Time) {
 	actionStr := strings.ToLower(action.String())
 	metrics.RunawayCheckerCounter.WithLabelValues(r.resourceGroupName, matchType, actionStr).Inc()
-	r.manager.markRunaway(r.resourceGroupName, r.originalSQL, r.planDigest, actionStr, matchType, now)
+	r.manager.markRunaway(r.resourceGroupName, r.originalSQL, r.planDigest, r.sqlDigest, actionStr, matchType, now)
```

Comment:
Consider passing `*Checker` directly to reduce the number of parameters.

Reply:
replaced